openedx/features/enterprise_support/api.py

```diff
@@ -786,6 +785,33 @@ def get_enterprise_learner_data_from_db(user):
         return serializer.data
 
 
+@enterprise_is_enabled(otherwise=[])
+def get_data_sharing_consents_from_db(user):
+    """
+    Returns a list of data sharing consent records for the given user.
+    """
+
+    return DataSharingConsent.objects.filter(
+        username=user.username
+    )
+
+
+@enterprise_is_enabled(otherwise=[])
+def get_enterprise_course_enrollments_from_db(user):
```

Comment:
nit: get_enterprise_course_enrollments. There is no need to have **from_db** at the end.